CMakeLists.txt

```diff
@@ -147,7 +150,7 @@ endif()
 
 set(MULL_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
-set (MULL_CXX_FLAGS "-std=c++14 -fPIC -fvisibility-inlines-hidden -fno-exceptions -Wunused -Wall -Werror")
+set (MULL_CXX_FLAGS "-fPIC -fvisibility-inlines-hidden -fno-exceptions -Wunused -Wall -Werror")
```

Comment:
`-fvisibility-inlines-hidden` can also be set in cmake via  [`CMAKE_VISIBILITY_INLINES_HIDDEN`](https://cmake.org/cmake/help/latest/variable/CMAKE_VISIBILITY_INLINES_HIDDEN.html)
-fPIC also has a cmake property but that one is usually set automatically when it is needed.

Reply:
This one conflicts with the json library (just shows some warnings).